source/tutorials/import-sequence-data.rst

```diff
@@ -0,0 +1,56 @@
+Importing sequence data
+=======================
+
+.. note:: This guide assumes you have performed the steps in the :doc:`install guide <../install>` and have read :doc:`Importing Data <./import>`.
+
+To begin a QIIME 2 analysis with sequence data, for example as generated on an Illumina MiSeq instrument, it must be imported into a QIIME 2 artifact (``.qza``) file. This document illustrates how to import data in various different starting formats, and will be expanded as additional relevant formats are identified. Each section below briefly describes a format of sequence data, provides commands to download example data files, and illustrates how to import data in that format.
+
+QIIME 2 is not designed for use with any specific sequence data format, but rather is intended to be general to many different data formats. If you need to import data in a format that is not covered here, please post to the `QIIME 2 Forum`_ for help.
+
+"Earth Microbiome Project protocol" multiplexed fastq
+-----------------------------------------------------
+
+Brief format description
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the "Earth Microbiome Project" protocol format, there are two ``fastq.gz`` files, one containing sequence reads and one containing the associated barcode reads, and the data are still multiplexed. The order of the records in the two ``fastq.gz`` files defines the association between a sequence read and its barcode read. These can be imported as follows.
+
+Obtaining example data
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. command-block::
+
+   mkdir raw-sequences
+   curl -sL https://data.qiime2.org/2.0.6/tutorials/moving-pictures/raw-sequences/barcodes.fastq.gz > raw-sequences/barcodes.fastq.gz
+   curl -sL https://data.qiime2.org/2.0.6/tutorials/moving-pictures/raw-sequences/sequences.fastq.gz > raw-sequences/sequences.fastq.gz
+
+Importing data
+~~~~~~~~~~~~~~
+
+.. command-block::
+
+   qiime tools import --type RawSequences --input-path raw-sequences/ --output-path raw-sequences
```

Comment:
Thanks. This one actually works if you include the `.qza` extension. That's what we're doing in the *Moving Pictures* tutorial, just made that change here (I think I must have edited it in parallel with testing yesterday, thinking that the extension wasn't necessary).